src/conspiracies/docprocessing/docprocessor.py

```diff
@@ -100,7 +111,7 @@ def process_docs(
                 (remove_context(doc._.resolve_coref), id_)
                 for doc, id_ in coref_resolved_docs
             ),
-            batch_size=batch_size * 4,
+            batch_size=self.batch_size * 4,
```

Comment:
*4?

Reply:
Collecting batches from the coref pipeline which uses insane amounts of memory on larger batches. I can give more details if needed.

Reply:
I would name the 4, so that we also in the future know this